src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        final int NUMBER_OF_BALLS = 3;
```

Comment:
It is not a constant
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
        private static final int NUMBER_OF_BALLS = 3;
```